src/OSPSuite.Core/Services/ICanCopyToClipboard.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace OSPSuite.Core.Services
+{
+   public interface ICanCopyToClipboard
```

Comment:
Those interfaces need to be implemented by all view and presenters that are not using the ChartDisplayPresenter